test/GPv2AccessControl.test.ts

```diff
@@ -3,16 +3,16 @@ import { Contract } from "ethers";
 import { ethers, waffle } from "hardhat";
```

Comment:
Since you're touching this file, can you rename it to match the contract: `GPv2AllowListAuthentication.test.ts`?